Modules/IO/RAW/wrapping/itkRawImageIO.wrap

```diff
@@ -1,5 +1,8 @@
 itk_wrap_class("itk::RawImageIO" POINTER)
+  UNIQUE(to_types "UC;${WRAP_ITK_SCALAR}")
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
-    itk_wrap_template("${ITKM_F}${d}" "${ITKT_F},${d}")
+    foreach(t ${to_types})
+      itk_wrap_template("${ITKM_VI${t}${d}}${ITKM_VI${t}${d}}" "${ITKT_VI${t}${d}},${ITKT_VI${t}${d}}")
```

Comment:
`VI` means VectorImage, and this is a rarely used type.
```suggestion
      itk_wrap_template("${ITKM_${t}${d}}${ITKM_${t}${d}}" "${ITKT_${t}${d}},${ITKT_${t}${d}}")
```

Reply:
Alternatively, use `itk_wrap_image_filter("${WRAP_ITK_SCALAR}" 2)` which will have about the same effect.